book/section3/wallets.md

```diff
@@ -1,12 +1,37 @@
 ---
 title: "Wallets"
 ---
+## Introduction
 
-A deep dive on what an Identus Identity wallet is and represents.  We will discuss seed phrases and how they work.
+Wallets are an essential component on every Self-Sovereign Identity interaction, and as you might have guessed, just like in the physical world a wallet holds your identifiers (IDs), in the digital SSI realm their function is to store and manage Decentralized Identifiers (DIDs), Verifiable Credentials (VCs), cryptographic keys, and other related assets.
```

Comment:
"Just link in the physical world `where` a wallet.....(IDs), `a` digital SSI `wallet's` function is to..."